vite.config.firefox.ts

```diff
@@ -0,0 +1,56 @@
+import { defineConfig, Plugin } from 'vite';
+import react from '@vitejs/plugin-react';
+import { crx } from '@crxjs/vite-plugin';
+import zipPack from 'vite-plugin-zip-pack';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+import fs from 'node:fs';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+import path from 'node:path';
+
+import manifest from './manifest.json';
+
+function myPlugin(): Plugin {
```

Comment:
This is a workaround for a chrome problem. I believe it is not necessary for firefox. (It's causing firefox to complain, so I would remove it)

Reply:
I wasn't sure about this at first, it's fixed now